client/src/SignIn.js

```diff
@@ -93,9 +93,15 @@ function SignIn({ config, refreshAppContext }) {
     </div>
   );
 
+  function createMarkupForSamlLink() {
+    return { __html: config.samlLinkHtml };
+  }
+
   const samlForm = (
     <div>
-      <a href={config.baseUrl + '/auth/saml'}>Sign in with ADFS</a>
+      <a href={config.baseUrl + '/auth/saml'}>
+        <span dangerouslySetInnerHTML={createMarkupForSamlLink()} />
```

Comment:
Can we either provide a default value for samlLinkHtml in config or a default button here if HTML is not provided?

Reply:
I have a default value in config-items.js. It's not enough?

Reply:
Ah sorry I had missed the text. I was skimming for something that looked like HTML - this works